FlexMeasures/flexmeasures
# [UI] Stop adding one day to the end of the range on the asset page

## How to test?

Open the asset page, select a period and refresh the page. The period should be the same.

## Related Items

Closes #1055 

---
